docs/development.rst

```diff
@@ -7,12 +7,15 @@ Source
 
 It's on `GitHub <https://github.com/mozilla/fathom>`_.
 
+Windows Considerations
+======================
+
+Fathom uses `Makefiles <https://www.gnu.org/software/make/manual/make.html>`_ to build its npm package, commandline tools, and docs; as well as run its tests. Additionally, these Makefiles rely on Unix commands. Therefore, if you are developing on Windows, you need access to these Unix commands through something like `Cygwin <https://www.cygwin.com/>`_. You can build and test Fathom using `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_, but just know that you are technically building and testing Fathom in Linux when you do.
```

Comment:
I think we could tersen: "Fathom uses makefiles to do its builds and run its tests. These rely on Unix commands. ..."

Reply:
And yes, we could probably move the Windows section to the bottom of the page, as it'll likely be used seldom.